Mitarbeiter.py

```diff
@@ -0,0 +1,78 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from Vorgesetzter import Vorgesetzter
+
+
+class Mitarbeiter:
+
+    _bestelllimit: int = 20
+    _vorgesetzter: Vorgesetzter = None
+
+    def __init__(self, name) -> None:
+        self._name: str = name
+
+    @classmethod
+    def setzeAllgemeinesLimit(cls, limit: int) -> int:
+        cls._bestelllimit: int = limit
+        return cls._bestelllimit
+
+    def setzeVorgesetzten(self, name: Vorgesetzter) -> Vorgesetzter:
+        self._vorgesetzter = name
+        return name
+
+    def setzeName(self, name: str) -> str:
+        self._name = name
+
+    def bekommeName(self) -> str:
+        return self._name
+
+    def bekommeVorgesetzten(self) -> Vorgesetzter:
+        if self._vorgesetzter == None and self.__class__.__name__ == "Mitarbeiter":
+            print("freier Mitarbeiter")
```

Comment:
```suggestion
            print("freier Mitarbeiter")
            return None
```

maybe? Otherwise the function won't return anything without explicitly stating it.

Reply:
+1

Also, the return type should reflect the possible `None` :)